attractions/rollup.config.js

```diff
@@ -1,20 +1,41 @@
 import svelte from 'rollup-plugin-svelte';
 import resolve from '@rollup/plugin-node-resolve';
+import autoPreprocess from 'svelte-preprocess';
 import pkg from './package.json';
 
 const name = pkg.name
   .replace(/^(@\S+\/)?(svelte-)?(\S+)/, '$3')
   .replace(/^\w/, m => m.toUpperCase())
   .replace(/-\w/g, m => m[1].toUpperCase());
 
-export default {
-  input: 'index.js',
-  output: [
-    { file: pkg.module, 'format': 'es' },
-    { file: pkg.main, 'format': 'umd', name },
-  ],
-  plugins: [
-    svelte(),
-    resolve(),
-  ],
-};
+export default [
+  {
+    input: 'index.js',
+    output: [
+      { file: pkg.module, 'format': 'es' },
+      { file: pkg.main, 'format': 'umd', name },
+    ],
+    plugins: [
+      svelte(),
+      resolve(),
+    ],
+  },
+  {
+    // This will likely replace the other configuration above
```

Comment:
Not sure if it's a good idea to replace the other configuration with this one. The `customElement` option alters the constructor in a way that might only be compatible with browsers that support custom elements. Perhaps we could somehow distribute two versions of Attractions, one that's suitable for custom elements and a regular one.

Reply:
I think the [support](https://caniuse.com/custom-elementsv1) is relatively fine. 94.05% of all users is not bad at all, so I wouldn't worry about older browsers.
Also, the other API is way more difficult, in addition to that it does not include the CSS (with `customElements`, they get inlined in the JS), removing any convenience that would come from using custom components (without Svelte) in general.

Reply:
Well, Safari is not supported, which is sad, but then again, as long as it is still possible to properly use the library with Svelte, this thing won't be a critical problem